### PR TITLE
Add setters in SiPMSensl class

### PR DESCRIPTION
**Pull request by halmamol**
This PR adds setters for `time_binning_` and `visibility_` parameters in SiPMSensl class. The NewKDB geometry is accordingly adapted.

### CODE REVIEW

**Comment by paolafer** [comment]
Some tests are failing because you removed the configuration parameters of the `SensL` class. Please, update the macros accordingly.

File: source/geometries/NextNewKDB.cc [hunk 1]
```diff
@@ -36,11 +36,18 @@ namespace nexus {
...
     /// Messenger
     msg_ = new G4GenericMessenger(this, "/Geometry/KDB/", "Control commands of kapton dice board geometry.");
     msg_->DeclareProperty("visibility", visibility_, "Kapton Dice Boards Visibility");
+
+    G4GenericMessenger::Command& time_binning_cmd = msg_->DeclareProperty("sipm_time_binning", time_binning_, "TP SiPMs time binning.");
+    time_binning_cmd.SetParameterName("sipm_time_binning", false);
+    time_binning_cmd.SetUnitCategory("Time");
+    time_binning_cmd.SetRange("sipm_time_binning>0.");
+    
     sipm_ = new SiPMSensl;
```
Comment (paolafer) [inline source/geometries/NextNewKDB.cc:54]: I would add also the `sipm_visibility` parameter and set it in the code.

**Comment by halmamol** [comment]
I've added `sipm_visibility` in NextNewKDB, and I've modified NEW macros accordingly including both parameters (`sipm_visibility` and `sipm_time_binning`).

**Review by paolafer — APPROVED** [review]
This PR adapts the `SiPMSensL` class to be used in the DEMO++ code. Messengers are removed to use this class as one of the possible sensor geometries to be instantiated in the DEMO SiPM boards, through a parameter.

**Pull request merged**